e2e/api/steps.js

```diff
@@ -72,6 +73,29 @@ module.exports = {
     await assertCorrectEventsAreAvailableToUser(config.defendantSolicitorUser, 'PROCEEDS_WITH_OFFLINE_JOURNEY');
   },
 
+  amendClaimDocuments: async () => {
+    eventName = 'ADD_OR_AMEND_CLAIM_DOCUMENTS';
+    let returnedCaseData = await apiRequest.startEvent(eventName, caseId);
+    assertContainsPopulatedFields(returnedCaseData);
+    caseData = returnedCaseData;
+
+    await validateEventPages(data[eventName]);
+
+    await assertCallbackError('Upload', data[eventName].invalid.Upload.duplicateError,
+      'More than one particular of claim added');
+
+    await assertCallbackError('Upload', data[eventName].invalid.Upload.nullError,
+      'One particular of claim is required');
+
+    await assertSubmittedEvent('AWAITING_CASE_NOTIFICATION', {
+      header: 'Documents uploaded successfully',
+      body: '<br />'
+    }, true);
+
+    await assertCorrectEventsAreAvailableToUser(config.solicitorUser, 'AWAITING_CASE_NOTIFICATION');
+    await assertCorrectEventsAreAvailableToUser(config.defendantSolicitorUser, 'AWAITING_CASE_NOTIFICATION');
```

Comment:
Aren't we missing a few here: 
![image](https://user-images.githubusercontent.com/138029/108046070-1f469480-703c-11eb-93de-45efbeb5861e.png)

Reply:
Same are missing in `createClaimWithRepresentedRespondent`.

Reply:
the assertion asserts for state, so we are asserting all these events are available 
```  
AWAITING_CASE_NOTIFICATION: [
    events.NOTIFY_DEFENDANT_OF_CLAIM,
    events.ADD_DEFENDANT_LITIGATION_FRIEND,
    events.CASE_PROCEEDS_IN_CASEMAN,
    events.ADD_OR_AMEND_CLAIM_DOCUMENTS
  ],
```